bench/Main.hs

```diff
@@ -1,9 +1,12 @@
+{-# OPTIONS_GHC -Wno-unused-imports -Wno-unused-top-binds #-}
```

Comment:
I’d prefer if we didn’t have unused imports or top-level bindings rather than silencing the warning. Dead code is dead, & all that.

Failing that, could you please 📝 the purpose behind silencing the warnings?

Reply:
Oh yeah, let me remove this. I think this is where some `cabal.project` changes to remove `ghc-options: -Werror` and add that into the CI build might be nice as it's painful to iterate during development with those settings. I'll remove this as it's not needed.